lib/account/account.controllers.js

```diff
@@ -50,6 +74,18 @@ module.exports = (models) => {
         return error;
       });
     },
+
+    patchAccount: async function(request, h) {
+    let account = request.payload;
+    let accountId = request.params.id;
+    let accountInstance = await getAccount(accountId);
+
+    // Make sure account exists
+    if (accountInstance === null) {
+      throw Boom.badRequest('Account does not exist');
+    }
+    return await accountInstance.update(account);
+  },
```

Comment:
this would allow anyone to edit anyone's account. One option is to only use the id from the request credentials (like we do when creating the account), or, still ask for the account id as a param and do an 'access check' where you make sure the id from the params matches the id from the credentials (this way would allow extending/modifying that check to also allow superadmin edit access).

Also, please indent this section so it lines up. Thanks!

Reply:
/ If we are forgoing access controls for MVP you can just leave a TODO note here.